app/models/storage_location.rb

```diff
@@ -175,11 +178,22 @@ def decrease_inventory(itemizable_array)
     log
   end
 
+  def verify_inventory_items
+    unless empty_inventory_items?
+      errors.add(:base, "Cannot delete storage location containing inventory items")
```

Comment:
```suggestion
      errors.add(:base, "Cannot delete storage location containing inventory items with non-zero quantities")
```
This is more specific since you can delete in cases the quantity is 0